src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
+@import "./reset.scss";
+@import "./blocks/variables.scss";
+@import "./blocks/header.scss";
+@import "./blocks/product-card.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/cards-container.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
```

Comment:
Isn't it already in your reset.scss file?

Reply:
it is, remove it from main.scss